src/Kunstmaan/MediaBundle/Validator/Constraints/IsExtensionAllowed.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Kunstmaan\MediaBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class IsExtensionAllowed extends Constraint
+{
+    public const NOT_ALLOWED = 1;
```

Comment:
I would do it the "symfony way" and use an uuid as unique message id. This id will be unique over all constraints.

See: https://github.com/symfony/symfony/blob/e6e1ca38c05d3c42780eb06bc98161ab4629adbf/src/Symfony/Component/Validator/Constraints/Blank.php#L24